docs/no_peeking/mappings/specimen.py

```diff
@@ -0,0 +1,72 @@
+def status(x):
+    """
+    http://hl7.org/fhir/R4/valueset-specimen-status.html
+    """
+    if x == constants.COMMON.FALSE:
+        return 'unavailable'
+    elif x == constants.COMMON.TRUE:
+        return 'available'
+    else:
+        raise Exception('Unknown Specimen status')
+
+def specimen_type(x):
+    """
+    http://hl7.org/fhir/R4/v2/0487/index.html
+    """
+    if x == constants.SPECIMEN.COMPOSITION.BLOOD:
+        return	{
+            'coding': 'BLD',
+            'display': 'Whole blood'
+        }
+    elif x == constants.SPECIMEN.COMPOSITION.SALIVA:
+        return	{
+            'coding': 'SAL',
+            'display': 'Saliva'
+        }
+    elif x == constants.SPECIMEN.COMPOSITION.TISSUE:
+        return	{
+            'coding': 'TISS',
+            'display': 'Tissue'
+        }
+    else:
+        raise Exception('Unknown Specimen type')
+
+
+specimen = {
+    'resourceType': 'Specimen',
+    'id': BIOSPECIMEN.TARGET_SERVICE_ID,
+    'biosample-individual-age-at-collection': None,
+    'biosampe-histological-diagnosis': None,
+    'biosample-tumor-progression': None,
+    'biosample-tumor-grade': None,
+    'biosample-control': True or False, # defaults to True
```

Comment:
can we get comments on some of these ambiguous ones?

Reply:
I will provide a more complete README.md describing my findings and rationale for this work as well as docstrings per every mapper.

Reply:
And could you clarify what "these ambiguous ones" means, @fiendish?

Reply:
so like it says "True or False" which is of course logically always True. what is the field actually supposed to have in it?

Reply:
Aha, I see what you mean. I will also add some inline comments across all the mappers if the assignment of a certain value is not clear.

Reply:
The required changes have been made!